src/firestore/collection/changes.ts

```diff
@@ -14,9 +12,10 @@ import { DocumentChangeAction, Action } from '../interfaces';
  */
 export function docChanges(query: Query): Observable<DocumentChangeAction[]> {
   return fromCollectionRef(query)
-    .map(action =>
-      action.payload.docChanges
-        .map(change => ({ type: change.type, payload: change })));
+    .pipe(
+      map(action =>
+        action.payload.docChanges
+          .map(change => ({ type: change.type, payload: change }))));
```

Comment:
shouldn't this be a pipe + map?

Reply:
action.payload.docChanges isn't Observable, so I think no?

Reply:
I see, derp.